### PR TITLE
Multi controllers

### PR DESCRIPTION
**Pull request by tmushayahama**
### Issues
 https://github.com/geneontology/pybiopax2gocam/issues/5

### Added

- Parsed controllers object for bpx file. still specific for R-HSA-204174 file 

### Changed
- data structure to accommodate list<Controllers>
- various places where previous controller and control_type was being used

### CODE REVIEW

File: src/models/parsers/reactome_parser.py [hunk 1]
```diff
@@ -93,19 +95,42 @@ def _process_bp(self, xrefs)->Term:
...
-    def process_controller(self, sp):
-        if isinstance(sp, pybiopax.biopax.Catalysis):
-            sp_uid = sp.controlled.uid             
-            self._control_map[sp_uid].append(sp.control_type)
-        
     
+    def _process_controllers(self, sp):
+        sp_uid = sp.controlled.uid    
+        for controller in ReactomeParser.get_object_list(sp.controller):                  
+            xrefs = [f'{v.db}:{v.id}' for v in controller.xref if v.db.lower() in ACCEPTED_DBS]
```
Comment (dustine32) [inline src/models/parsers/reactome_parser.py]: Some other desired IDs are also buried in `controller.entity_reference.xrefs`. We should start extracting these too.

**Review by dustine32 — CHANGES_REQUESTED** [review]
@tmushayahama This branch totally accomplishes the purpose of handling multiple controllers per reaction.

There's another issue of extracting the right ID's for these controllers. Many have a `ChEBI` ID hidden not in the `controller.xrefs` but instead in `controller.entity_reference.xrefs`. A specific example is controller NADH on `BiochemicalReaction3`. I added a test that curently fails but will pass once the correct ChEBI ID is found. But this could be a new ticket/branch if you want.

You can run the test from the cmd line with: `python -m pytest`

File: src/models/parsers/reactome_parser.py (hunk 1 above)
Comment (tmushayahama) [reply src/models/parsers/reactome_parser.py]: @dustine32 I have made changes, seems working, but the test is wrong as we need to check  using lowercase

**Review by dustine32 — APPROVED** [review]
Thanks @tmushayahama for fetching xrefs from the entity references now!

**Pull request merged**